generics/constants/common.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * name : constants/common.js
+ * author : Akash Shah
+ * Date : 09-Dec-2019
+ * Description : All common messages.
+ */
+
+
+module.exports = {
+  "ACTIVE": "active",
+  "INACTIVE": "inactive",
+  "TRUE": true,
+  "FALSE": false,
+  schema: {},
+  "RESPONSE_OK": "OK",
+  "PUBLIC_ROLE": "PUBLIC"
+
```

Comment:
@rakeshSgr - Remove extra lines and schema if not required.

Reply:
resolved